conf/application.conf

```diff
@@ -173,4 +173,8 @@ constants {
 
 beta {
   serviceName = "changeDestination"
+}
+
+auditing {
+  enabled = true
```

Comment:
Other frontends either don't have this set (it defaults to `false` through Bootstrap), or it's set to `false` manually locally - was this left `true` by mistake?

Reply:
Yes, that was a mistake, fixed on https://github.com/hmrc/emcs-tfe-change-destination-frontend/pull/53/commits/8ca7e26997e221c5f801c318122711ae9316c355